android/app/src/main/java/org/fossasia/openevent/core/bookmark/BookmarkStatus.java

```diff
@@ -1,16 +1,73 @@
 package org.fossasia.openevent.core.bookmark;
 
-import lombok.AllArgsConstructor;
-import lombok.Data;
-
-@Data
-@AllArgsConstructor
 public class BookmarkStatus {
 
     private int storedColor;
     private int sessionId;
     private Status actionCode;
 
+    public BookmarkStatus(int storedColor, int sessionId, Status actionCode) {
```

Comment:
Can be converted to data class

Reply:
I did convert this into data class. My local repo has this as a data class but when I push it, its coming out as a normal pojo.